build/TargetFramework.cs

```diff
@@ -10,6 +10,7 @@
 public class TargetFramework : Enumeration
 {
     public static readonly TargetFramework NET462 = new() { Value = "net462" };
+    public static readonly TargetFramework NETCore3_1 = new() { Value = "netcoreapp3.1" };
```

Comment:
Double-checking: the fact that this targeted was added to the enumeration doesn't mean that we will build any other sources or tests for netcoreapp3.1, right?

Reply:
Correct. It is not used with any other source or tests.